### PR TITLE
TAS-36/Credit_Card_Model

### PR DESCRIPTION
**Pull request by Brendan-bx**

### CODE REVIEW

File: apps/api/prisma/schema.prisma [hunk 1]
```diff
@@ -34,3 +36,16 @@ model Service {
   createdAt                  DateTime @default(now())
   updatedAt                  DateTime @updatedAt
 }
+
+model Credit_Card {
```
Comment (Limerio) [inline apps/api/prisma/schema.prisma]: Je ne veux pas de 🐍 case

File: apps/api/prisma/schema.prisma [hunk 2]
```diff
@@ -34,3 +36,16 @@ model Service {
   createdAt                  DateTime @default(now())
   updatedAt                  DateTime @updatedAt
 }
+
+model creditCard {
```
Comment (Limerio) [inline apps/api/prisma/schema.prisma]: a model needs to start with a capital letter

File: apps/api/prisma/schema.prisma [hunk 3]
```diff
@@ -19,6 +19,8 @@ model User {
   createdAt       DateTime  @default(now())
   updatedAt       DateTime  @updatedAt
 
+  creditCard CreditCard[]
```
Comment (Limerio) [inline apps/api/prisma/schema.prisma]:
```suggestion
  creditCards CreditCard[]
```

File: apps/api/prisma/schema.prisma [hunk 4]
```diff
@@ -34,3 +36,16 @@ model Service {
   createdAt                  DateTime @default(now())
   updatedAt                  DateTime @updatedAt
 }
+
+model CreditCard {
+  id             String   @id @default(uuid())
+  userId         String
+  user           User     @relation(fields: [userId], references: [id])
```
Comment (Limerio) [inline apps/api/prisma/schema.prisma]:
```suggestion
  user           User     @relation(fields: [userId], references: [id], onDelete: Cascade)
```

**Review by Limerio — APPROVED** [review]

**Review by Limerio — APPROVED** [review]

**Review by Limerio — APPROVED** [review]

**Review by WookiesRpeople2 — APPROVED** [review]

**Pull request merged**